core/server/proxy/src/main/java/alluxio/proxy/PathsRestServiceHandler.java

```diff
@@ -144,7 +141,7 @@ public Integer call() throws Exception {
         }
         return mStreamCache.put(os);
       }
-    }, ServerConfiguration.global());
+    }, ServerConfiguration.global(), null);
```

Comment:
is adding the last argument `null` here necessary?

Reply:
No,it is should be removed because the the overloaded call method with callable and alluxioConf param is already passing null to the call method (with headers params as Nullable) in RestUtils.
Thanks